tests/integration/commands/test_pack.py

```diff
@@ -31,24 +34,112 @@
     [
         textwrap.dedent(
             """\
-            name: my-bundle
+            # name: my-bundle
```

Comment:
This is xfail, so we do not need to comment out it?

Reply:
You caught me with something half-done! It's only xfail on Windows, but I was testing with and without the name and forgot to parametrise it to two tests.I've fixed that now.

Thanks for catching this!